packages/components/src/internal/components/files/FileAttachmentContainer.tsx

```diff
@@ -22,7 +22,9 @@ import { Map, Set } from 'immutable';
 
 import { fileMatchesAcceptedFormat, fileSizeLimitCompare } from './actions';
 import { FileAttachmentEntry } from './FileAttachmentEntry';
-import { ALL_FILES_LIMIT_KEY, FileSizeLimitProps } from './models';
+import { ALL_FILES_LIMIT_KEY } from './models';
+import {FileSizeLimitProps} from "../../..";
```

Comment:
should this be importing from `../../../public/files/models`?

Reply:
Following Alan's general rule mentioned in Frontend chat, keeping it as-is. Also, if i change it, it is not building for me anyway.
_- If it is a sibling file (in the same directory as me) import it (e.g. import { thing } from './thing')
- If it is a child file (in a child directory) don't import it, refactor it so it's a sibling, or elsewhere
- If it is above me import from index
- else: import directly (e.g. it's not in our index.ts)_